vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -211,6 +211,65 @@ public FormLayout() {
     public FormLayout(Component... components) {
         add(components);
     }
+    
+    /**
+     * Sets the colspan of the given component's element.
+     * You can directly add components with the wanted colspan with
+     * {@link #add(Component)}.
```

Comment:
>	     * {@link #add(Component)}.
This should probably link to `#add(Component, int)` instead ?

Reply:
You're right.